src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.report;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport(Map<String, Integer> storage) {

```

Comment:
The `getReport` method should not start with an empty line. Please remove the empty line above the method signature.